db/migrate/20170816143355_add_expense_columns_to_snap_application.rb

```diff
@@ -0,0 +1,29 @@
+class AddExpenseColumnsToSnapApplication < ActiveRecord::Migration[5.1]
+  def change
+    add_column :snap_applications, :rent_expense, :integer
+    add_column :snap_applications, :property_tax_expense, :integer
+    add_column :snap_applications, :insurance_expense, :integer
+    add_column :snap_applications, :utility_heat, :boolean
+    add_column :snap_applications, :utility_cooling, :boolean
+    add_column :snap_applications, :utility_electrity, :boolean
+    add_column :snap_applications, :utility_water_sewer, :boolean
+    add_column :snap_applications, :utility_trash, :boolean
+    add_column :snap_applications, :utility_phone, :boolean
+    add_column :snap_applications, :utility_other, :boolean
+
+    add_column :snap_applications, :dependent_care, :boolean
+    add_column :snap_applications, :medical, :boolean
```

Comment:
I feel like a bool field called `medical` is not super clear -- thoughts on a more descriptive name for this column? (feel similarly about `dependent_card` `court_ordered` and `tax_deductible`)